community/values/src/main/java/org/neo4j/values/Values.java

```diff
@@ -37,11 +40,16 @@
 @SuppressWarnings( "WeakerAccess" )
 public class Values
 {
+    public static final Value MIN_NUMBER = Values.doubleValue( Double.NEGATIVE_INFINITY );
+    public static final Value MAX_NUMBER = Values.doubleValue( Double.NaN );
+    public static final Value MIN_STRING = Values.stringValue( "" );
+    public static final Value MAX_STRING = Values.booleanValue( false );
+
     private Values()
     {
     }
 
-    interface ValueLoader<T>
+    public interface ValueLoader<T>
```

Comment:
would be really nice to this class and `ValueLoadException` in their own classes isntead